app/assets/javascripts/discourse/controllers/tags-show.js

```diff
@@ -155,7 +156,18 @@ export default Controller.extend(BulkTopicSelection, FilterModeMixin, {
     },
 
     changeTagNotificationLevel(notificationLevel) {
-      this.tagNotification.update({ notification_level: notificationLevel });
+      this.tagNotification
+        .update({ notification_level: notificationLevel })
+        .then(response => {
+          User.currentProp(
```

Comment:
Because this is a controller I would prefer if you used `this.currentUser` to get the user rather than the `User` singleton. Saves an import and makes testing easier due to dependency injection.